### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by danielabbatt**
Merging recent changes into main

### CODE REVIEW

File: Meraki.Api/Data/WirelessRfProfileCreateUpdateRequest.cs [hunk 1]
```diff
@@ -11,54 +11,54 @@ public class WirelessRfProfileCreateUpdateRequest
...
-	public MinBitrateType MinBitrateType { get; set; }
+	public MinBitrateType? MinBitrateType { get; set; }
 
 	/// <summary>
 	/// Gets or Sets BandSelectionType
 	/// </summary>
 	[ApiAccess(ApiAccess.ReadWrite)]
 	[DataMember(Name = "bandSelectionType")]
-	public BandSelectionType BandSelectionType { get; set; }
+	public BandSelectionType? BandSelectionType { get; set; }
```
Comment (devel0peter) [inline Meraki.Api/Data/WirelessRfProfileCreateUpdateRequest.cs:35]: This being nullable allows one to create a Wireless Rf Profile without setting BandSelectionType, which - according to the documentation - is a required parameter during creation.

Comment (danielabbatt) [reply Meraki.Api/Data/WirelessRfProfileCreateUpdateRequest.cs:35]: As agreed with @devel0peter, we're going to merge this in but ideally we need 2 separate models.

Comment (devel0peter) [reply Meraki.Api/Data/WirelessRfProfileCreateUpdateRequest.cs:35]: As agreed with @danielabbatt it's OK for now, as the model has been improved. We will move towards separating Creation and Update models, including this one.

**Review by devel0peter — APPROVED** [review]
Looks good, great improvements on the Test project!

**Pull request merged**